rem/aubuf/aubuf.c

```diff
@@ -265,13 +272,20 @@ int aubuf_append_auframe(struct aubuf *ab, struct mbuf *mb,
 {
 	struct frame *f;
 	size_t sz;
+	int err;
 
 	if (!ab || !mb)
 		return EINVAL;
 
-	f = mem_zalloc(sizeof(*f), frame_destructor);
-	if (!f)
-		return ENOMEM;
+	struct mem_pool_entry *e = mem_pool_borrow(ab->pool);
+	if (!e) {
+		err = mem_pool_extend(ab->pool, 10);
```

Comment:
I am not sure: Should the borrow function do the extend implicitely?

Reply:
I replaced this with: `mem_pool_borrow_extend`